pkgs/development/tools/build-managers/bazel/bazel_4/default.nix

```diff
@@ -617,7 +617,7 @@ stdenv.mkDerivation rec {
       ./bazel_src/output/bazel-complete.fish
   '';
 
-  doInstallCheck = true;
+  doInstallCheck = stdenv.hostPlatform.system != "aarch64-darwin";
```

Comment:
This should have a comment why it is disabled on aarch64-darwin.

Reply:
Good point, @uri-canva do you want to add it?

Reply:
Done: https://github.com/NixOS/nixpkgs/pull/145613.